packages/web-react/src/components/Modal/demo/ModalHiddenCloseButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useToggle } from '../../../hooks';
+import { Button } from '../../Button';
+import Modal from '../Modal';
+import ModalBody from '../ModalBody';
+import ModalDialog from '../ModalDialog';
+import ModalFooter from '../ModalFooter';
+import ModalHeader from '../ModalHeader';
+
+const ModalHiddenCloseButton = () => {
+  const [isOpen, isOpenToggle] = useToggle(false);
+  const handleClose = () => isOpenToggle();
+
+  return (
+    <>
+      <Button onClick={isOpenToggle}>Open Modal</Button>
+
+      <Modal
+        id="example-hidden-close-button"
+        isOpen={isOpen}
+        onClose={handleClose}
+        closeOnBackdropClick={false}
+        closeOnEscapeKeyDown={false}
+      >
+        <ModalDialog>
+          <ModalHeader id="example-hidden-close-button" hasCloseButton={false}>
```

Comment:
do we need the id? If so, please make it unique

Reply:
we don't, id removed